src/main/java/edu/cornell/kfs/module/purap/CUPurapParameterConstants.java

```diff
@@ -54,4 +54,6 @@ public static class ElectronicInvoiceParameters {
     public static final String JAGGAER_WEBSERVICE_UPLOAD_SUPPLIER_PASSWORD = "UPLOAD_SUPPLIER_PASSWORD";
     
     public static final String MAX_FILE_SIZE_PO_SEND_TO_VENDOR = "MAX_FILE_SIZE_PO_SEND_TO_VENDOR";
+    
+    public static final String IWNT_DOC_ENABLE_IWANT_CONTRACT_TAB_IND_PARM = "ENABLE_IWANT_CONTRACT_TAB_IND";
```

Comment:
Since this is in a Parameter Constants class, there's no need for the "\_PARM" suffix in the constant's name. Also, since it's obvious from the parameter name that it's meant for the IWant Document, you could potentially remove the "IWNT\_DOC\_" prefix from the constant's name as well.

Reply:
updated